lib/deaf_grandma.rb

```diff
@@ -14,13 +14,27 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      if @bye_counter > 0
```

Comment:
@chivygab The spec says to say "SEE YOU LATER SONNY!" and exit after yelling 3 times, not just the first time.